src/main.rs

```diff
@@ -233,11 +273,17 @@ fn run_test_case_inner(
     case: &TestCase,
     extra_client_args: &Vec<String>,
     extra_server_args: &Vec<String>,
+    extra_shared_args: &Vec<String>,
 ) -> Result<(std::process::Output, std::process::Output), i32> {
     // Create the server and client args
     let mut server_args = extra_server_args.clone();
     let mut client_args = extra_client_args.clone();
 
+    for arg in extra_shared_args {
+        server_args.push(arg.clone());
+        client_args.push(arg.clone());
```

Comment:
Surely there is a better way to concatenate two vectors, perhaps `server_args.append(extra_shared_args.clone())`.

Reply:
Indeed there is a better way do this. I fixed it.